onnxruntime/contrib_ops/cuda/bert/group_query_attention_helper.h

```diff
@@ -19,7 +19,7 @@
                    void* parameters,
                    int num_heads,
                    int kv_num_heads,
-                   const Tensor* past_seq_len,
+                   const Tensor* attention_mask,
                    bool is_past_bsnh,
```

Comment:
>                  bool is_past_bsnh, [](http://example.com/codeflow?start=1&length=36)

as we only support bnsh, you can clean up the code

Reply:
i thought i'd leave it since everything is in place to support bsnh